### PR TITLE
release: sourcegraph@5.3.0

### PR DESCRIPTION
**Pull request by keegancsmith**
This pull request is part of the Sourcegraph 5.3.0 release.

* [Release batch change](https://sourcegraph.sourcegraph.com/organizations/sourcegraph/batch-changes/release-sourcegraph-5.3.0)
* [Tracking issue](https://github.com/sourcegraph/sourcegraph/issues/60032)

### Test plan

CI checks in this repository should pass, and a manual review should confirm if the generated changes are correct.

### CODE REVIEW

File: docs/CHANGELOG.mdx
Comment (BolajiOlajide) [inline docs/CHANGELOG.mdx:2899]: This needs to be reverted. Unfortunately, we'll need to ignore this file next time.

**Review by BolajiOlajide — APPROVED** [review]
Approving to unblock. Please don't merge yet, there's something I need to manually revert here.

File: docs/CHANGELOG.mdx
Comment (keegancsmith) [inline docs/CHANGELOG.mdx:2899]: This change looks incorrect.

```suggestion
- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.11.1` will need to manually modify their `docker run` command to use `sourcegraph/server:3.11.4` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```

@BolajiOlajide any idea how to stop the release tool from touching this file?

File: docs/admin/executors/deploy_executors_kubernetes.mdx
Comment (keegancsmith) [inline docs/admin/executors/deploy_executors_kubernetes.mdx:102]: lol this is clearly wrong and was previously updated automatically.
```suggestion
- Disable access to `169.254.169.254` (AWS / GCP Instance Metadata Service).
```

File: docs/CHANGELOG.mdx
Comment (keegancsmith) [reply docs/CHANGELOG.mdx:2899]: snap, also notice the other place with badness I picked up on.

Comment (BolajiOlajide) [reply docs/CHANGELOG.mdx]: Just pushed this to stop that: https://github.com/sourcegraph/sourcegraph/pull/60490

The second is a bit tricky, maybe?

**Pull request merged**
